aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  Layer,
+  Form,
+  Footer,
+  FormField,
+  Heading,
+  Spinner,
+  StarRating,
+  ResponsiveContext,
+  Text,
+  TextArea,
+} from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+const FeedbackLayout = ({ children, onClick, onClose }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  return (
+    // feedback layout
+    <Layer>
+      <Box
+        fill="vertical"
+        overflow="auto"
+        width={!['xsmall', 'small'].includes(breakpoint) ? 'medium' : undefined}
+        pad="medium"
+        gap="medium"
+      >
+        <Heading level={4} size="small" margin="none">
+          Feedback on powering devices
+        </Heading>
+        {children}
+        <Footer direction="row" justify="end" gap="small">
+          <Button onClick={onClose} label="Cancel" />
+          <Button
+            onClick={onClick}
+            label="Submit Feedback"
+            primary
+            type="submit"
+          />
+        </Footer>
+      </Box>
+    </Layer>
+  );
+};
+
+export const PowerOnExample = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [rating, setRating] = useState();
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (isSuccessful) {
+        setShowFeedback(true);
+      }
+    }, 1000);
+  }, [isSuccessful]);
+
+  // simulating a call to a power on API call. Demo purposes
+  const PowerOnLoading = () => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      setIsSuccessful(true);
+    }, 3000);
+  };
+
+  // fetch call to POST api should be in onSubmit
+  // for demo purposes disabling actual API call
+  // submitFeedback();
```

Comment:
I don't know if this is necessary. I think adding this https://github.com/grommet/hpe-design-system/pull/3076/files#r1063863588 will take care of it.